scripts/main/album.js

```diff
@@ -43,6 +43,10 @@ album.getID = function () {
 
 };
 
+album.isTagAlbum = function() {
+	return album.json['tag_album'] === '1';
```

Comment:
Is there a reason you are using array syntax rather than `album.json.tag_album`? This is not a trick question; I'm genuinely curious :smiley:.

Reply:
there is no reason. Dot variant is more pretty, you are right.